src/Pipfile

```diff
@@ -9,6 +9,9 @@ importlib-metadata = "*"
 pyahocorasick = "*"
 
 [dev-packages]
+tqdm = "*"
```

Comment:
We already have a progress meter. Please use that one rather than introducing another dependency